sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationkeyvaultoptions.py

```diff
@@ -3,27 +3,51 @@
 # Licensed under the MIT License. See License.txt in the project root for
```

Comment:
Should we rename the file?

Reply:
Do you have a suggestion?

Reply:
maybe _azure_appconfiguration_keyvault_options.py?

@annatisch how do you think?

Reply:
Moved to `_models.py`